main.nf

```diff
@@ -448,6 +451,19 @@ workflow {
 
     // STEP 1: MAPPING READS TO REFERENCE GENOME WITH BWA-MEM
 
+    //TODO: not sure whether to split the reads before qc or after
```

Comment:
I think so far we were splitting before QC.
Not sure what would be the best option...

Reply:
Not sure either. I will fix the merging first, that is where it fails currently and then think about wether it should be moved up or not.

Reply:
we can ask the community about it actually

Reply:
Hm, if we do it before the QC, QC would run on each pair individually. Are the reports merged?

Reply:
Not sure we can merge reports.
And it would make sense to have the QC on the fastqs we're starting with actually